### ISSUE TITLE
Add support for `destroy` in Key-Value Engine

### ISSUE THREAD

**Issue by ps-burgstaller** [quarkiverse/quarkus-vault#77]
Hi team,

I am trying to completely remove a key-value pair from Vault, including removing the path. The `delete` method on the KVEngine only deactivates the secret, as far as I understood it. 

I suppose the `destroy` method would allow us to completely remove a secret from Vault. Is support for this method planned? I'd help out, but I would need some pointers on where to do this.

Kind regards!

**Comment by kdubb** [comment]
I am not sure what you mean "deactivates" a secret. A secret is store at a "path". Deleting a secret is akin to deleting its path.

**Comment by kdubb** [comment]
Sorry, I forgot that deleted KV secrets can be undeleted for a period of time before they are fully removed completely.

**Comment by vsevel** [comment]
yes there is a [destroy](https://www.vaultproject.io/api-docs/secret/kv/kv-v2#destroy-secret-versions) operation.
do you want to try out creating a PR @ps-burgstaller ?

**Comment by ps-burgstaller** [comment]
I'm gonna take a look at it. 👍

**Comment by vsevel** [comment]
hello @ps-burgstaller . if this is still of interest for you, please look at the proposed PR, and tell us what you think. thanks.
